.code-samples.meilisearch.yaml

```diff
@@ -1038,3 +1038,5 @@ date_guide_sort_1: |
       "q": "",
       "sort": ["release_timestamp:desc"]
     }'
+instance_options_config_file_download_1: |
+  curl https://raw.githubusercontent.com/meilisearch/meilisearch/main/config.toml > config.toml
```

Comment:
Unless I'm mistaken, instance options cannot be configured with an SDK, so perhaps it's better to leave it outside the code sample component?

@brunoocasali, what do you think?

Reply:
Yes, that's correct! The SDK uses an already-initialized instance :)